python/src/aac/lang/constants.py

```diff
@@ -17,6 +17,7 @@
 # Common Definition Field Names
 DEFINITION_FIELD_ADD = "add"
 DEFINITION_FIELD_ARGUMENTS = "arguments"
+DEFINITION_FIELD_KW_ARGUMENTS = "kw_arguments"
```

Comment:
I think it may be better to keep this following the Python "standard" where we say:

```suggestion
DEFINITION_FIELD_ARGUMENTS = "args"
DEFINITION_FIELD_KW_ARGUMENTS = "kwargs"
```

Alternatively, although less desirable IMO, would be to spell everything out completely if we're going to start off with separate fields for an "args" tuple - i.e. arguments - and an "args" dictionary - i.e. keyword_arguments.